src/main/java/org/gridsuite/study/server/service/NotificationService.java

```diff
@@ -120,15 +120,6 @@ public void emitStudyChanged(UUID studyUuid, UUID nodeUuid, String updateType, S
                 .build());
     }
 
-    @PostCompletion
```

Comment:
remove UPDATE_TYPE_STUDY_DELETE constant to avoid future confusion

Reply:
done